src/content/docs/en/guides/content-collections.mdx

```diff
@@ -1,643 +1,181 @@
-  }),
-});
+- You have multiple files or data to organize that share the same overall structure (e.g. blog posts written in Markdown which all have the same frontmatter properties).
+- You have existing content stored remotely, such as in a CMS, and want to take advantage of the collections helper functions and Content Layer API instead of using `fetch()` or SDKs.
+- You need to fetch (tens of) thousands of related pieces of data, and need a querying and caching method that handles at scale.
 
-// 3. Export a single `collections` object to register your collection(s)
-export const collections = {
-  'blog': blogCollection,
-};
-```
+### When not to create a collection
 
-### Defining multiple collections
-
-You can use `defineCollection()` as many times as you want to create multiple schemas.  All collections must be exported from inside the single `collections` object.
-
-```ts
-// src/content/config.ts
-const blogCollection = defineCollection({
-  type: 'content',
-  schema: z.object({ /* ... */ })
-});
-const newsletter = defineCollection({
-  type: 'content',
-  schema: z.object({ /* ... */ })
-});
-const authors = defineCollection({
-  type: 'data',
-  schema: z.object({ /* ... */ })
-});
-
-export const collections = {
-  'blog': blogCollection,
-  'newsletter': newsletter,
-  'authors': authors,
-};
-```
+Collections provide excellent structure, safety, and organization when you have **multiple pieces of content that must share the same properties**.
 
-As your project grows, you are also free to reorganize your codebase and move logic out of the `src/content/config.ts` file. Defining your schemas separately can be useful for reusing schemas across multiple collections and sharing schemas with other parts of your project.
-
-```ts
-// src/content/config.ts
-// 1. Import your utilities and schemas
-import { defineCollection } from 'astro:content';
-import { blogSchema, authorSchema } from '../schemas';
-
-// 2. Define your collections
-const blogCollection = defineCollection({
-  type: 'content',
-  schema: blogSchema,
-});
-const authorCollection = defineCollection({
-  type: 'data',
-  schema: authorSchema,
-});
-
-// 3. Export multiple collections to register them
-export const collections = {
-  'blog': blogCollection,
-  'authors': authorCollection,
-};
-```
+Collections **may not be your solution** if:
 
-### Using third-party collection schemas
+- You have only one or a small number of different pages. Consider [making individual page components](/en/basics/astro-pages/) such as `src/pages/about.astro` with your content directly instead.
+- Your data is not stored in a supported format, such as TOML. Use other methods of [importing files](/en/guides/imports/#import-statements) or [fetching data](/en/guides/data-fetching/) instead.
+- You are displaying files that are not processed by Astro, such as PDFs. Place these static assets in the [`public/` directory](/en/basics/project-structure/#public) of your project instead.
```

Comment:
- You are using APIs that need to be updated in real time. Content collections are only updated at build time, so if you need live data you should consider on-demand rendering / SSR

Reply:
It might be good to have a callout with this point as well, maybe in the bit about custom loaders

Reply:
added to the section about loaders in the defining collections page, for now